src/main/java/aj/Aj.java

```diff
@@ -0,0 +1,179 @@
+package aj;
+
+import java.util.Scanner;
+import java.io.IOException;
+
+
+/**
+ * Aj chat-bot class.
+ */
+public class Aj {
+
+    Parser parser;
+    Storage storage;
+
+    TaskList taskList;
+
+    Ui ui;
+
+
+    /**
+     * Request input from user and does the respective actions.
+     *
+     * @param scanner Use same scanner object for repeated function calls.
+     * @return true to exit program, else function will be continuously called.
+     * @throws NoSuchCommandException    If command from user input does not exist.
+     * @throws EmptyDescriptionException If second part of user input does not exist.
+     * @throws IndexOutOfRangeException  If user gives an index > taskList size.
+     * @throws IOException               Arose if there is issue updating database.
+     */
+    public boolean canAskCommand(Scanner scanner) throws NoSuchCommandException, EmptyDescriptionException,
+            IndexOutOfRangeException, IOException {
```

Comment:
Perhaps abstract this part of the code out into another file, like Parser instead?